src/main/java/org/logstash/beats/Server.java

```diff
@@ -27,19 +27,21 @@
     private final int port;
     private final NioEventLoopGroup workGroup;
     private final String host;
+    private final int beatsHeandlerThread;
```

Comment:
Nit: Might want to make this `beatsHandlerThreadCount`

Reply:
This one my tooling would have flagged. Especially `fchurn` (which only reports new errors)